packages/react/src/ProgressBar/ProgressBar.tsx

```diff
@@ -57,36 +55,65 @@ const ProgressContainer = styled.span<StyledProgressContainerProps>`
   ${sx};
 `
 
-export type ProgressBarProps = React.HTMLAttributes<HTMLSpanElement> & {bg?: string} & StyledProgressContainerProps &
-  ProgressProp
+export type ProgressBarItems = React.HTMLAttributes<HTMLSpanElement> & {'aria-label'?: string} & ProgressProp & SxProp
 
-export const ProgressBar = forwardRef<HTMLSpanElement, ProgressBarProps>(
-  (
-    {animated, progress, bg = 'success.emphasis', barSize = 'default', children, ...rest}: ProgressBarProps,
-    forwardRef,
-  ) => {
-    if (children && progress) {
-      throw new Error('You should pass `progress` or children, not both.')
+export const Item = forwardRef<HTMLSpanElement, ProgressBarItems>(
+  ({progress, 'aria-label': ariaLabel, ...rest}, forwardRef) => {
+    const progressAsNumber = typeof progress === 'string' ? parseInt(progress, 10) : progress
+
+    const ariaAttributes = {
+      'aria-valuenow': progressAsNumber && progressAsNumber >= 0 ? Math.round(progressAsNumber) : undefined,
+      'aria-valuemin': 0,
+      'aria-valuemax': 100,
     }
 
     warning(
-      children &&
+      !ariaAttributes['aria-valuenow'] &&
         typeof (rest as React.AriaAttributes)['aria-valuenow'] === 'undefined' &&
         typeof (rest as React.AriaAttributes)['aria-valuetext'] === 'undefined',
       'Expected `aria-valuenow` or `aria-valuetext` to be provided to <ProgressBar>. Provide one of these values so screen reader users can determine the current progress. This warning will become an error in the next major release.',
     )
```

Comment:
I'm curious if there's a reason why we didn't include `ariaAttributes['aria-valuenow']` as a value to include in the `warning` conditional? Since we automatically create the `aria-valuenow` based on `progressAsNumber` being true and a number, is that sufficient enough if consumers don't provide an `aria-valuenow` themselves? Also, I added `progressAsNumber >= 0` so we can include `aria-valuenow` values that start at 0.

Cc: @kendallgassner - would love your insight, but I realize that it's been a minute since we've introduced these changes 😅

Reply:
huh, your right that is odd and if a user passed it in it would override the value, we are setting via `...rest`. 🤔 I think it's safe to remove the warning all together.